<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item alignment="Qt::AlignLeft">
         <widget class="QLineEdit" name="lineEdit_IP">
          <property name="text">
           <string>127.0.0.1</string>
          </property>
          <property name="placeholderText">
           <string>ip-adddres</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Connect">
          <property name="text">
           <string>connect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_10">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="line_9">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="pushButton_StreamCam">
          <property name="minimumSize">
           <size>
            <width>91</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>stream_cam</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="InitOpenCV_button">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>codec</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_codec">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>H264</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>JPEG</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>bitrate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_Bitrate">
          <property name="maximum">
           <number>5000</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>500</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item alignment="Qt::AlignLeft">
         <widget class="QComboBox" name="comboBox_Mode">
          <item>
           <property name="text">
            <string>eAlarmBlue                   </string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>eAlarmBlueHi</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>eGrayBW</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>eGrayWB</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>eAlarmGreen</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>eIron</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>eIronHi</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>eMedical</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>eRainbow</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>eAlarmRed</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_7">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string> Temperature min-max </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_temperatureMIN">
            <property name="minimum">
             <number>-20</number>
            </property>
            <property name="maximum">
             <number>999</number>
            </property>
            <property name="value">
             <number>-20</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_temperatureMAX">
            <property name="maximum">
             <number>999</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_ApplyTemperatureRange">
            <property name="text">
             <string>Apply</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="Line" name="line_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_8">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>mode scale </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_Scale">
            <item>
             <property name="text">
              <string>0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_FOV">
          <property name="text">
           <string>FOV</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_FOV">
          <property name="value">
           <number>75</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_Shutter">
          <property name="text">
           <string>Shutter</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="LabelVideo" name="labelOpenCVFrame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(136, 138, 133);</string>
      </property>
      <property name="text">
       <string>--------------------------------------------------------------- | Video Stream |  ----------------------------------------------------------------</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LabelVideo</class>
   <extends>QLabel</extends>
   <header>labelvideo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
